docker-versions-maven-plugin/src/main/java/com/github/cafapi/docker_versions/plugins/DockerVersionsMojo.java

```diff
@@ -47,4 +47,9 @@ protected String getProjectDockerRegister()
             PROJECT_DOCKER_REGISTRY,
             project.getArtifactId() + "-" + project.getVersion() + ".project-registries.local");
     }
+
+    protected String getDockerHost()
+    {
+        return project.getProperties().getProperty("docker.host");
```

Comment:
The Docker Maven Plugin has a `dockerHost` configuration for this:
https://github.com/fabric8io/docker-maven-plugin/blob/v0.44.0/src/main/java/io/fabric8/maven/docker/AbstractDockerMojo.java#L134-L135

Shouldn't we create have the same?